runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -0,0 +1,35 @@
+" Debugger commands.
```

Comment:
It is for Vim8 script.
I don't think it works on neovim.

It should be included?

Reply:
It's harmless to include it. I think we should adapt it to work with Nvim, eventually.

Reply:
OK.